.github/workflows/build-lint-test.yml

```diff
@@ -45,3 +45,46 @@ jobs:
         run: cargo clippy --all-targets --all-features -- -D warnings
       - name: Run tests
         run: cargo test --verbose
+  node:
+    name: Node
```

Comment:
Maybe:
```suggestion
    name: Frontend
```

Reply:
And Backend for the backend job, but that's outside of the diff so I can't suggest it.

Reply:
Changed both the key and the `name` of both jobs.